components/HeaderBlock.vue

```diff
@@ -1,13 +1,20 @@
 <template>
-  <header class="container flex-row-reverse py-4 md:flex md:flex-row">
-    <div>
-      <client-only>
-        <RadioPlayer :show-album-art="true" :now-playing-uri="streamServer" />
-      </client-only>
+  <header class="flex-row-reverse items-center justify-between px-4 py-4 bg-white md:flex md:flex-row">
+    <div class="flex items-center">
+      <div class="block mr-4 cursor-pointer" @click="$router.push('/')">
+        <img src="/img/lahmacun-logo.png" alt="" class="w-24">
+      </div>
+      <div class="mr-8">
+        <client-only>
+          <RadioPlayer :show-album-art="true" :now-playing-uri="streamServer" />
+        </client-only>
+      </div>
     </div>
-    <div class="flex items-center cursor-pointer" @click="$router.push('/')">
-      <img src="/img/lahmacun-logo.png" alt="" class="h-16 mr-4">
-      <h1>Lahmacun Radio</h1>
+    <HeadNavigation />
+    <div id="time">
```

Comment:
I like your logic of having HeaderBlock > HeaderNavigation 

Is there a reason time is not a component of its own?

Reply:
could be a component on it's own for sure